data/constants/keycodes/extras/keycodes_ergol_0.0.1.hjson

```diff
@@ -0,0 +1,570 @@
+        "S(EG_7)": {
+            "key": "EG_AMPR",
+            "label": "&",
+        }
+        "S(EG_8)": {
+            "key": "EG_ASTR",
+            "label": "*",
+        }
+        "S(EG_9)": {
+            "key": "EG_HASH",
+            "label": "#",
+        }
+        "S(EG_0)": {
+            "key": "EG_AT",
+            "label": "@",
+        }
+        "S(EG_SLSH)": {
+            "key": "EG_MINS",
+            "label": "-",
+        }
+        "S(EG_EQ)": {
+            "key": "EG_PLUS",
+            "label": "+",
+        }
+        "S(EG_1DK)": {
+            "key": "EG_EXLM",
+            "label": "!",
+        }
+        "S(EG_LBRC)": {
+            "key": "EG_LCBR",
+            "label": "{",
+        }
+        "S(EG_RBRC)": {
+            "key": "EG_RCBR",
+            "label": "}",
+        }
+        "S(EG_QUOT)": {
+            "key": "EG_DQUO",
+            "label": "\"",
+        }
+        "S(EG_BSLS)": {
+            "key": "EG_PIPE",
+            "label": "|",
+        }
+        "S(EG_LABK)": {
+            "key": "EG_RABK",
+            "label": ">",
+        }
+        "S(EG_DOT)": {
+            "key": "EG_COLN",
+            "label": ":",
+        }
+        "S(EG_COMM)": {
+            "key": "EG_SCLN",
+            "label": ";",
+        }
+
+/* AltGr symbols
+ * ┌───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───────┐
+ * │   │ ₁ │ ₂ │ ₃ │ ₄ │ ₅ │ ₆ │ ₇ │ ₈ │ ₉ │ ₀ │   │   │       │
+ * ├───┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─────┤
+ * │     │ ^ │ < │ > │ $ │ % │ @ │ & │ * │ ' │ ` │   │   │     │
+ * ├─────┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┐    │
+ * │      │ { │ ( │ ) │ } │ = │ \ │ + │ - │ / │ " │   │   │    │
+ * ├────┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴───┴────┤
+ * │    │   │ ~ │ [ │ ] │ _ │ # │ | │ ! │ ; │ : │ ? │          │
+ * ├────┼───┴┬──┴─┬─┴───┴───┴───┴───┴───┴──┬┴───┼───┴┬────┬────┤
+ * │    │    │    │                        │    │    │    │    │
+ * └────┴────┴────┴────────────────────────┴────┴────┴────┴────┘
+ */
+        "ALGR(EG_1)": {
+            "key": "EG_SUB_1",
+            "label":  "₁",
+        }
+        "ALGR(EG_2)": {
+            "key": "EG_SUB_2",
+            "label":  "₂",
+        }
+        "ALGR(EG_3)": {
+            "key": "EG_SUB_3",
+            "label":  "₃",
+        }
+        "ALGR(EG_4)": {
+            "key": "EG_SUB_4",
+            "label":  "₄",
+        }
+        "ALGR(EG_5)": {
+            "key": "EG_SUB_5",
+            "label":  "₅",
+        }
+        "ALGR(EG_6)": {
+            "key": "EG_SUB_6",
+            "label":  "₆",
+        }
+        "ALGR(EG_7)": {
+            "key": "EG_SUB_7",
+            "label":  "₇",
+        }
+        "ALGR(EG_8)": {
+            "key": "EG_SUB_8",
+            "label":  "₈",
+        }
+        "ALGR(EG_9)": {
+            "key": "EG_SUB_9",
+            "label":  "₉",
+        }
+        "ALGR(EG_0)": {
+            "key": "EG_SUB_0",
+            "label":  "₀",
```

Comment:
Are the U+202F `NARROW NO-BREAK SPACE` characters after `"label":` here and elsewhere in the file really intended? I am not sure if all (H)JSON parsers accept whitespace characters other than U+20 `SPACE`.

Reply:
No, I asked an AI assistant to generate them for me, and obviously it formatted it as if it was text. The parser doesn't understand this, see lines 100.. of the header file. Thanks for spotting this !